zebra-network/src/address_book.rs

```diff
@@ -463,6 +514,13 @@ impl AddressBook {
         self.by_addr.descending_values().cloned()
     }
 
+    /// Returns true if there is a recently live peer connection with the provided IP address.
```

Comment:
```suggestion
    /// Returns true if there is a recently live outbound peer connection with the provided IP address.
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/7029/commits/0c769b657fbe1f66dd20b7ed874ce4b529395395.